content/news/2018/05/brand-bible-new-merchants/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Brand Bible & New Merchants"
+date: 2018-05-02T18:18:43+12:00
+draft: false
+author: "NavCore"
+description: ""
+resources: []
+categories: ["Nav News"]
+slug: ""
+type: "news"
+feature_image: "imghero.jpg"
+show_title_text: false
+---
+
+The great thing about decentralized community projects like this is that freedom of the individual is paramount. No one can tell you what NavCoin is or isn't, it is what you make it. This week there have been so many great contributions from all around the world where people are starting to put this empowering idea into practice.
+<!--more-->
+### What's happening in the community
+
+NavCoin Italia reports on Reddit that a specialist medical clinic in Rome has begun accepting NavCoin. NavCoin Italia has been very active lately - so make sure to follow them and keep up with all they’re doing.
+
+<br />
+<section style="text-align: center">
+![](img4.png)
+<br /><br />
+</section>
+
+Meanwhile at the other end of the retail spectrum check out NavCoin being accepted by these enterprising crypto enthusiasts at a flea market in Germany. Using NavPay they’ll see their payment come through almost instantly. Have you seen anything similar in your town? 
+
+<br />
+<section style="text-align: center">
+![](img1.png)
+<br /><br />
+</section>
+
+Jonathon (on Discord) has been experimenting with button designs using the NavCoin colour palate and different angles at the sides. It was great to see so much positive feedback from other members of the community. Here is just one of Jonathon’s options.
+
+<br />
+<section style="text-align: center">
+![](img2.png)
+<br /><br />
+</section>
+
+Beekart (on Discord) has been giving his design and illustration skills a good workout, creating banners featuring original hand drawn characters. What’s more in the spirit of our open-source decentralised community Beekart has made his work freely available to the community. Check it out on the NavCoin page on Reddit.
+
+<br />
+<section style="text-align: center">
+![](img3.png)
+<br /><br />
+</section>
+
+### Meet up in the Netherlands
+The details for the upcoming meetup on 12 May have changed. Here is the latest:
+The new location is The Florin in Utrecht starting at 7 p.m. You can view the location at [www.florinutrecht.nl](https://www.florinutrecht.nl). Hopefully there will be plenty of photos to share with you after the event.
+
+### Technical Updates
+The updated NavPay release is getting closer. There have been some interesting challenges around handling multi-server clusters, but the developers working on it say to look out for an announcement soon, once testing and bug-fixing is complete.
+
+Work on the update to the paper wallet is progressing well, with the Windows version next on the list. The devnet is almost ready too and it will be used to test upcoming pull requests for NavCoin Core. An alternative block explorer is being built which can used for the devnet and testnet.
```

Comment:
Should we seperate each idea out into it's own paragraph?

`Work on the update to the paper wallet is progressing well, with the Windows version next on the list.`

`The devnet is almost ready too and it will be used to test upcoming pull requests for NavCoin Core.`

`An alternative block explorer is being built which can used for the devnet and testnet.`